### PR TITLE
New French translation done by Leetram_519

### PR DESCRIPTION
**Pull request by Parexy**
The old one was apparently made with google translate so Leetram_519 was kind enough to provide us with a new one made without it.

### CODE REVIEW

File: Languages/French/Keyed/Multiplayer.xml [hunk 1]
```diff
@@ -2,109 +2,131 @@
...
   <MpChatButton>Chat</MpChatButton>
-  <MpTradingButton>Commerce</MpTradingButton>
+  <MpTradingButton>Echanges</MpTradingButton>
   
   <MpChatPlayers>Joueurs ({0}):</MpChatPlayers>
   <MpSend>Envoyer</MpSend>
   <MpSteamAcceptTitle>Accepter les joueurs Steam</MpSteamAcceptTitle>
-  <MpSteamAcceptDesc>Cliquez pour accepter</MpSteamAcceptDesc>
-  <MpSteamAccepted>Joueur accepté</MpSteamAccepted>
+  <MpSteamAcceptDesc>Clicker pour accepter</MpSteamAcceptDesc>
```
Comment (Mihokyo) [inline Languages/French/Keyed/Multiplayer.xml]:
```suggestion
  <MpSteamAcceptDesc>Cliquez pour accepter</MpSteamAcceptDesc>
```
"Cliquez" was correct

File: Languages/French/Keyed/Multiplayer.xml [hunk 2]
```diff
@@ -2,109 +2,131 @@
...
   <MpDownloading>Téléchargement ({0}%)</MpDownloading>
   
-  <MpSteamConnectingTo>Connection à une partie hébergée par {0}</MpSteamConnectingTo>
-  <MpSteamConnectingWaiting>En attente de la confirmation de l'hôte</MpSteamConnectingWaiting>
+  <MpSteamConnectingTo>Connexion à une partie hébergée par {0}</MpSteamConnectingTo>
+  <MpSteamConnectingWaiting>En attente de l'acceptation de l'hote</MpSteamConnectingWaiting>
   
-  <MpTicksBehind>Vous avez {0} ticks de retard.</MpTicksBehind>
-  <MpLowerGameSpeed>Envisagez de diminuer la vitesse du jeu.</MpLowerGameSpeed>
+  <MpTicksBehind>Tu es {0} ticks en retard.</MpTicksBehind>
```
Comment (Mihokyo) [inline Languages/French/Keyed/Multiplayer.xml]:
```suggestion
<MpTicksBehind>Tu as {0} ticks de retard.</MpTicksBehind>
```
Tu as = You have
Tu es = You are

File: Languages/French/Keyed/Multiplayer.xml [hunk 3]
```diff
@@ -2,109 +2,131 @@
...
+  <MpLowerGameSpeed>Baissez la vitesse de jeu.</MpLowerGameSpeed>
+  <MpForceCatchUp>Clickez pour rattraper votre retard.</MpForceCatchUp>
```
Comment (Mihokyo) [inline Languages/French/Keyed/Multiplayer.xml]:
```suggestion
<MpForceCatchUp>Cliquez pour rattraper votre retard.</MpForceCatchUp>
```

File: Languages/French/Keyed/Multiplayer.xml [hunk 4]
```diff
@@ -2,109 +2,131 @@
...
   <MpDirect>Directe</MpDirect>
   <MpSteam>Steam</MpSteam>
   <MpHostTab>Héberger</MpHostTab>
   
   <MpMultiplayer>Multijoueur</MpMultiplayer>
-  <MpSingleplayer>Solo</MpSingleplayer>
+  <MpSingleplayer>Jeu solo</MpSingleplayer>
+  
+  <MpNotAvailable>Pas disponible en multijoueur.</MpNotAvailable>
+  <MpCallingFactionNotAvailable>Appeller d'autres factions est impossible en multijoueur.</MpCallingFactionNotAvailable>
```
Comment (Mihokyo) [inline Languages/French/Keyed/Multiplayer.xml]:
```suggestion
<MpCallingFactionNotAvailable>Appeler d'autres factions est impossible en multijoueur.</MpCallingFactionNotAvailable>
```
Gramatical:
Appeller => Appeler

File: Languages/French/Keyed/Multiplayer.xml [hunk 5]
```diff
@@ -2,109 +2,131 @@
...
   
- <MpNotAvailable>Non disponible en multijoueur.</MpNotAvailable>
- <MpCallingFactionNotAvailable>Appeler d'autres factions est actuellement impossible en Multijoueur.</MpCallingFactionNotAvailable>
+  <MpAggressiveTicking>Tick agressif</MpAggressiveTicking>
+  <MpAggressiveTickingDesc>Rattrapper le retard sur le serveur, au prix de quelques FPS.</MpAggressiveTickingDesc>
```
Comment (Mihokyo) [inline Languages/French/Keyed/Multiplayer.xml]:
```suggestion
<MpAggressiveTickingDesc>Rattraper le retard sur le serveur, au prix de quelques FPS.</MpAggressiveTickingDesc>
```

Gramatical:
Rattrapper => Rattraper

File: Languages/French/Keyed/Multiplayer.xml [hunk 6]
```diff
@@ -2,109 +2,131 @@
...
   
-  <MpReplayOutdated>(Obsolète)</MpReplayOutdated>
-  <MpReplayOutdatedDesc1>Ce replay était compatible avec la version du protocole {0}, mais vous avez la version {1}.</MpReplayOutdatedDesc1>
-  <MpReplayOutdatedDesc2>Le replay peut ne pas jouer correctement.</MpReplayOutdatedDesc2>
-  <MpReplayOutdatedDesc3>Les incompatibilitées possibles sont listées dans l'historique des versions.</MpReplayOutdatedDesc3>
+  <MpReplayOutdated>(Outdated)</MpReplayOutdated>
+  <MpReplayOutdatedDesc1>Cette sauvegarde a été faite dans la version {0}, mais vous utilisez la version {1}.</MpReplayOutdatedDesc1>
+  <MpReplayOutdatedDesc2>Cette sauvegarde n'est pas valide.</MpReplayOutdatedDesc2>
+  <MpReplayOutdatedDesc3>Les incompatibilités possibles sont sur le github.</MpReplayOutdatedDesc3>
+  <MpReplayOutdatedDesc4>This file contains additional information and hosting from it should work fine.</MpReplayOutdatedDesc4>
```
Comment (Mihokyo) [inline Languages/French/Keyed/Multiplayer.xml]:
```suggestion
  <MpReplayOutdatedDesc4>Ce fichier contient des informations supplémentaires, l'hébergement depuis celui-ci devrait fonctionner correctement.</MpReplayOutdatedDesc4>
```

File: Languages/French/Keyed/Multiplayer.xml [hunk 7]
```diff
@@ -2,109 +2,131 @@
...
-  <MpMaxPlayers>Joueurs maximum</MpMaxPlayers>
-  <MpAutosaveEvery>Fréquence sauvegarde auto</MpAutosaveEvery>
-  <MpAutosaveDays>journées</MpAutosaveDays>
-  <MpLanDesc1>Diffuser la partie sur votre réseau local.</MpLanDesc1>
-  <MpLanDesc2>Adresse LAN déterminée: {0}</MpLanDesc2>
-  <MpArbiterDesc>Une instance du jeu tournant dans le fond et qui aide avec les problèmes de désynchronisation.</MpArbiterDesc>
-  <MpAsyncTimeDesc>Sépare les controles du temps for chaque map et la planète.</MpAsyncTimeDesc>
-  <MpExperimentalFeature>Fonctionnalitée expérimentale</MpExperimentalFeature>
+  <MpMaxPlayers>Joueurs Max</MpMaxPlayers>
+  <MpAutosaveEvery>Autosauvegarde toute les</MpAutosaveEvery>
```
Comment (Mihokyo) [inline Languages/French/Keyed/Multiplayer.xml]:
```suggestion
  <MpAutosaveEvery>Sauvegarde automatique toute les</MpAutosaveEvery>
```

**Review by MilanDierick — APPROVED** [review]

**Review by SamboyCoding — APPROVED** [review]

**Pull request merged**